js/pages/data-sources/classes/Treemap.js

```diff
@@ -34,12 +36,18 @@ define([
 			this.byValueAsConcept = false;
 			this.byOperator = false;
 			this.byQualifier = false;
-
+			this.handleConceptClick = node => {
+				if (authApi.isPermittedViewDataSourceReportDetails(this.context.currentSource().sourceKey)) {
```

Comment:
This will block all report details when authentication is turned off

Reply:
No need to check if auth turned off cause this check internally used in isPermittedViewDataSourceReportDetails function